packages/discussions/src/types.ts

```diff
@@ -244,8 +255,8 @@ export interface IWithSharing {
  * @export
  * @interface IWithSorting
  */
-export interface IWithSorting {
-  sortBy: string;
+export interface IWithSorting<SortEnum> {
```

Comment:
@tomwayson 

Only tagging you on this b/c this is a breaking change and I know we typically try to bundle together a bunch of breaking changes before we do a major bump. Curious though if this practice may have died w/ the introduction of semantic release though..